assets/js/guides.js

```diff
@@ -137,4 +137,25 @@ $(document).ready(function () {
     }
   });
 
+  $("#guides-subscribe-email").focus(() => {
+    $("#guides-subscribe-email").css("border", "1px solid #07a7fc");
+  });
+
+  $("#guides-subscribe-email").blur(() => {
+    $("#guides-subscribe-email").css("border", "none");
+  });
+
+  //Submits only when email is valid.
+  $("#newsletter-subscribe").click(e => {
+    e.preventDefault();
+
+    let email = $("#guides-subscribe-email");
+    if (email.is(":invalid")) {
+      $("#guides-subscribe-email").css("border", "1px solid red");
+    } else {
+      $("#guides-newsletter-form").submit();
+    }
+    return false;
```

Comment:
Does this function need to return something?

Reply:
Actually no it doesn't need to return anything. I've taken off the return statement